src/applications/personalization/connected-accounts/containers/ConnectedAcctApp.jsx

```diff
@@ -27,18 +31,25 @@ class ConnectedAcctApp extends React.Component {
     this.props.deleteConnectedAccount(accountId);
   };
 
+  dismissAlert = accountId => {
+    this.props.dismissDeletedAccountAlert(accountId);
+  };
+
   render() {
     let connectedAccountsView;
     if (this.props.loading) {
       connectedAccountsView = (
         <LoadingIndicator message="Loading your connected accounts..." />
       );
-    } else if (this.props.accounts.length > 0) {
+    } else if (
+      this.props.accounts.filter(account => !account.deleted).length > 0
+    ) {
```

Comment:
Can we extract the left side of this conditional expression into a function in order to follow the 1-concept-per-line principle? Alternatively, switching to an `else` block in which the left side of that conditional was assigned to a new local variable would also be an improvement IMO.